### PR TITLE
Custom rustfmt.toml settings for better auto-formatting

### PR DESCRIPTION
**Pull request by majorz**
Connects-To: #62

### CODE REVIEW

File: src/connection.rs [hunk 1]
```diff
@@ -21,11 +21,13 @@ impl Connection {
     fn init(dbus_manager: &Rc<DBusNetworkManager>, path: &str) -> Result<Self, String> {
         let settings = dbus_manager.get_connection_settings(path)?;
 
-        Ok(Connection {
-               dbus_manager: dbus_manager.clone(),
-               path: path.to_string(),
-               settings: settings,
-           })
+        Ok(
+            Connection {
```
Comment (cmr) [inline src/connection.rs]: Hm, I'm not a fan of this particular style decision, I prefer to the original. What do you think?

File: src/connection.rs [hunk 2]
```diff
@@ -64,18 +66,14 @@ impl Connection {
         match state {
             ConnectionState::Activated => Ok(ConnectionState::Activated),
             ConnectionState::Activating => {
-                wait(self,
-                     ConnectionState::Activated,
-                     self.dbus_manager.method_timeout())
-            }
+                wait(self, ConnectionState::Activated, self.dbus_manager.method_timeout())
```
Comment (cmr) [inline src/connection.rs]: This is a huge improvement!

File: src/connection.rs [hunk 3]
```diff
@@ -216,7 +207,7 @@ pub fn get_connections(dbus_manager: &Rc<DBusNetworkManager>) -> Result<Vec<Conn
 
 
 pub fn get_active_connections(dbus_manager: &Rc<DBusNetworkManager>)
-                              -> Result<Vec<Connection>, String> {
+    -> Result<Vec<Connection>, String> {
```
Comment (cmr) [inline src/connection.rs]: Not sure I'm a fan of this one, although at least the arrow makes it easier to differentiate return type from the body.

File: src/connection.rs (hunk 1 above)
Comment (majorz) [inline src/connection.rs]: Yes, that particular part felt odd to me as well when I added it. I will check which option did that and what else it involved.

File: src/connection.rs (hunk 3 above)
Comment (majorz) [inline src/connection.rs]: I will check this one as well.

Comment (majorz) [inline src/connection.rs]: This one is caused by `fn_args_layout = "Block"`, which positively affects function declarations with more arguments, but in this particular case not so.

File: src/connection.rs (hunk 1 above)
Comment (majorz) [inline src/connection.rs]: This one is caused by `fn_call_style = "Block"`, but improvements already landed in rustfmt master, so we will have better formatting for those cases soon.

**Pull request merged**